webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
+             double                        consentTotalRoundTripTime;
```

Comment:
Pre-consent there can only be a single RTT sample, so might as well use all data in the roundTripTime estimate.